cpmpy/solvers/ortools.py

```diff
@@ -64,24 +66,11 @@ def __init__(self, cpm_model=None, solver=None):
             raise Exception("Install the python 'ortools' package to use this '{}' solver interface".format(self.name))
         from ortools.sat.python import cp_model as ort
 
-        super().__init__(cpm_model, solver=solver)
-        self.name = "ortools"
-
-        if cpm_model is None:
-            self.user_vars = []
-            self.ort_model = ort.CpModel()
-            self.varmap = dict() # cppy var -> solver var
-        else:
-            # store original vars and objective (before flattening)
-            self.user_vars = get_variables_model(cpm_model)
-
-            # create model (includes conversion to flat normal form)
-            self.ort_model = self.make_model(cpm_model)
-
-        # create the solver instance
-        # (so its params can still be changed before calling solve)
```

Comment:
please keep this comment about why we also create ort_solver here, and not just internally in solve() which would also have been an option, except that you can then only changes its parameters/access it inside solve so already creating it here is more general.